.github/workflows/publish_wiki.yml

```diff
@@ -0,0 +1,22 @@
+name: "Push docs to wiki"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/wiki/**"
+  workflow_dispatch:
+
+jobs:
+   build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Upload Documentation to Wiki
+        uses: SwiftDocOrg/github-wiki-publish-action@v1
```

Comment:
Need to verify if we're OK with using this action.

The repository has been archived (hence, unmaintained) - but the action seems fairly stable and works (I've tested it aseparately).

Reply:
@timkpaine @trallard - What do you think and prefer here?

Reply:
@pavithraes action seems fine